services/web/server/src/simcore_service_webserver/socketio/module_setup.py

```diff
@@ -15,7 +15,7 @@
 
 
 @app_module_setup(
-    "simcore_service_webserver.socketio", ModuleCategory.SYSTEM, logger=log
+    "simcore_service_webserver.socketio", ModuleCategory.ADDON, logger=log
```

Comment:
this one I do agree. this can be an ADDON since this is something that does not prevent the webserver from functioning

Reply:
this is not the case for the webserver-GC . AFAIK it is not needed there, right?